tests/resources/example_robot_output.xml

```diff
@@ -272,6 +322,7 @@
 <stat pass="2" fail="0">JUNIT_ROBOT_TAG</stat>
 <stat pass="1" fail="0">NO_OXYGEN_HERE</stat>
 <stat pass="2" fail="0">ZAP_ROBOT_TAG</stat>
+<stat pass="2" fail="0">MY_DUMMY_HANDLER_ROBOT_TAG</stat>
```

Comment:
In the output (log.html) does not show this statistic. Is the output.xml malformed?